result/event.yaml

```diff
@@ -26,9 +26,9 @@ components:
           x-constraint:
           - '/components/schemas/Port/properties/name'
           - '/components/schemas/Flow/properties/name'
-          - '/components/schemas/Device.Bgpv4Route/properties/name'
-          - '/components/schemas/Device.Bgpv6Route/properties/name'
-          example: Bgpv4 Route Range 1
+          - '/components/schemas/Router.Bgpv4Route/properties/name'
```

Comment:
is this right?  or is it Bgp.V4Route?

Reply:
@SuryyaKrJana can you do a sweep of all other constraints as well (if this was indeed a problem)?  I think it will impact code generation/validation.

Reply:
did this change again?  BGP.V4RouteRange?  another sweep?

Reply:
No. last changed history showing the following.
 - '/components/schemas/Bgp.V4RouteRange/properties/name'
 - '/components/schemas/Bgp.V6RouteRange/properties/name'